pallets/spaces/src/benchmarking.rs

```diff
@@ -0,0 +1,87 @@
+//! Spaces pallet benchmarking.
```

Comment:
Format this file using our rustfmt configuration.
It will basically sort imports and add a new line at the end of file, but still :)